alpha/lib/model/DeliveryProfileVodPackagerHls.php

```diff
@@ -57,15 +67,17 @@ protected function buildHttpFlavorsArray()
 		}
 
 		$parentFlavors = parent::buildHttpFlavorsArray();
-		// Un-Comment code to add support for mixed (h264/h265/av1) playback
-		// $assetsRequireFMP4layback = VodPackagerDeliveryUtils::doAssetsRequireFMP4Playback($this->params->getflavorAssets());
-		// if($assetsRequireFMP4layback)
-		// {
-		// 	foreach ($parentFlavors as &$parentFlavor)
-		// 	{
-		// 		$parentFlavor['url'] = "container/fmp4/" . $parentFlavor['url'];
-		// 	}
-		// }
+		
+		$dpSupportFmp4Playback = $this->getSupportFmp4();
+		$assetsRequireFMP4layback = VodPackagerDeliveryUtils::doAssetsRequireFMP4Playback($this->params->getflavorAssets());
```

Comment:
typo assetsRequireFMP4layback missing p in the word playbcak

Reply:
Done